container/functional/keylime_agent_container-basic-attestation/test.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+        # update /etc/keylime.conf
+        limeBackupConfig
+        #getting ip of host
+        SERVER_IP=$(hostname -I | xargs)
+
+        # tenant
+        rlRun "limeUpdateConf tenant require_ek_cert False"
+        rlRun "limeUpdateConf tenant verifier_ip $SERVER_IP"
+        rlRun "limeUpdateConf tenant registrar_ip $SERVER_IP"
+
+        #registrar
+        rlRun "limeUpdateConf registrar ip $SERVER_IP"
+
+        #verifier
+        rlRun "limeUpdateConf verifier ip $SERVER_IP"
+        rlRun "limeUpdateConf verifier registrar_ip $SERVER_IP"
+
+        rlRun "limeStartVerifier"
+        rlRun "limeWaitForVerifier"
+        rlRun "limeStartRegistrar"
+        rlRun "limeWaitForRegistrar"
+        
+        CONT_NETWORK_NAME="agent_network"
+        rlRun "limeconCreateNetwork ${CONT_NETWORK_NAME} 172.18.0.0/16"        
+        rlRun "limeUpdateConf agent registrar_ip '\"$SERVER_IP\"'"
+
+        #container image build and preparation
+        IMAGE="agent_image"
+        rlRun "limeconPrepareAgentImage ${IMAGE}"
+        TESTDIR_FIRST=$(limeCreateTestDir)
+        TESTDIR_SECOND=$(limeCreateTestDir)
+        rlRun "echo -e '#!/bin/bash\necho ok' > $TESTDIR_FIRST/good-script.sh && chmod a+x $TESTDIR_FIRST/good-script.sh"
+        rlRun "echo -e '#!/bin/bash\necho ok' > $TESTDIR_SECOND/good-script.sh && chmod a+x $TESTDIR_SECOND/good-script.sh"
+        #allow container access to /dev/tpmrm0
+        rlRun "chmod o+rw /dev/tpmrm0"
```

Comment:
I think this is OK for now but I am wondering if we can do it differently when sharing the device with the container, i.e. use some UID mapping. Probably something to figure out later.